src/index.html

```diff
@@ -3,11 +3,320 @@
+                  alt="Phone (1)"
+                  class="product__img"
+                >
+                </a>
+
+                <h3 class="product__title">Phone (1)</h3>
+
+                <p class="product__description">Designed with instinct, to bring joy back to the everyday. Through the Glyph Interface, a perfected OS and exceptional dual camera. All startlingly fast.</p>
+
+                <span class="product__price">$ 499.00</span>
+              </article>
+
+              <article class="product recommended__product">
+                <a href="#" class="product__link">
+                  <img
+                  src="./images/product/ear-2.png"
+                  alt="Ear (2)"
+                  class="product__img"
+                >
+                </a>
+
+                <h3 class="product__title">Ear (2)</h3>
+
+                <p class="product__description">Perfecting the path from artist to ear. So that music sounds just as it was intended.</p>
+
+                <span class="product__price">$ 149.00</span>
+              </article>
+
+              <article class="product recommended__product">
+                <a href="#" class="product__link">
+                  <img
+                  src="./images/product/ear-stick.png"
+                  alt="Ear (stick)"
+                  class="product__img"
+                >
+                </a>
+                <h3 class="product__title">Ear (stick)</h3>
+
+                <p class="product__description">Tech you can’t feel. Leaving just space for sound to come to life. Powered by custom sound technology.</p>
+
+                <span class="product__price">$ 119.00</span>
+              </article>
+            </div>
+          </section>
+
+          <section class="categories">
+            <h2 class="categories-title">Browse Nothing <br> products by category</h2>
+
+            <div class="categories__wrapper">
+              <article class="category">
+                <div class="category__links">
+                  <a href="#" class="category__link category__link--rectangle">
+                    <img
+                      src="./images/category/all-products-rectangle.png"
+                      alt="all products"
+                      class="category__img">
+                  </a>
+
+                  <a href="#" class="category__link category__link--square">
+                    <img
+                      src="./images/category/all-products-square.png"
+                      alt="all products"
+                      class="category__img">
+                  </a>
+                </div>
+
+                <h3 class="category__title">All products</h3>
+              </article>
+
+              <article class="category">
+                <div class="category__links">
+                  <a href="#" class="category__link category__link--square">
+                    <img
+                      src="./images/category/audio-square.png"
+                      alt="audio"
```

Comment:
![image](https://github.com/mate-academy/layout_landing-page/assets/54860641/c8ea9c1f-40de-435e-8f34-889d43913508)
Fix this in all code.